tools/go-agent/instrument/plugins/rewrite/rewrite.go

```diff
@@ -131,6 +131,13 @@ func (c *Context) rewriteTopLevelNames(files map[*FileInfo]*dst.File) {
 			case *dst.ImportSpec:
 			case *dst.TypeSpec:
 			case *dst.GenDecl:
+				if n.Tok == token.CONST && cursor.Parent() == f {
```

Comment:
Could you just combine the token verify between `CONST` and `VAR`? These codes are generally the same.